pallets/runtime/tests/src/staking/mod.rs

```diff
@@ -5819,3 +5819,31 @@ fn test_bond_too_small() {
         assert_ok!(both(90, 91, MinimumBond::get() + 1));
     });
 }
+
+#[test]
+fn validator_unbonding() {
+    ExtBuilder::default()
+        .validator_count(8)
+        .minimum_validator_count(1)
+        .build()
+        .execute_with(|| {
+            // 50 stash and 51 controller
+            bond(50, 51, 500000);
+            let entity_id = Identity::get_identity(&50).unwrap();
+
+            // Add a new validator successfully.
+            bond_validator_with_intended_count(50, 51, 500000, Some(2));
```

Comment:
The `bond` call isn't needed here, since the `bond_validator_with_intended_count` function calls `bond` too.